src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig

```diff
@@ -5,75 +5,99 @@
 
 <section>
     {{ form(form_content_location_add, {'action': path('ezplatform.location.add')}) }}
-    {% include '@ezdesign/ui/table_header.html.twig' with {
-        header_text: 'tab.locations.content_locations'|trans|desc('Content Locations'),
-        tools: tab.table_header_tools(form_content_location_add, form_content_location_remove, can_add)
+    {% set body_rows = [] %}
+    {% if locations is not empty %}
+        {% for location in locations %}
+            {% set body_row_cols = [] %}
+
+            {% set col_raw %}
+                {{ form_widget(form_content_location_remove.locations[location.id], {'attr': {'disabled': not location.canDelete}}) }}
+            {% endset %}
+            {% set body_row_cols = body_row_cols|merge([{
+                has_checkbox: true,
+                content: col_raw,
+            }]) %}
+
+            {% set col_raw %}
+                {% include '@ezdesign/ui/location_path.html.twig' with {'locations': location.pathLocations, 'link_last_element': true} %}
+            {% endset %}
+            {% set body_row_cols = body_row_cols|merge([{
+                content: col_raw,
+            }]) %}
+
+            {% set body_row_cols = body_row_cols|merge([
+                { content: location.childCount },
+            ]) %}
+
+            {% set col_raw %}
+                <label 
+                    class="ez-checkbox-icon {{ not location.explicitlyHidden ? 'is-checked' }}{% if not can_hide[location.id] %} disabled{% endif %}"
+                    title="{{ 'tab.locations.visibility'|trans|desc('Visibility') }}"
+                >
+                    <svg class="ibexa-icon ibexa-icon--view">
+                        <use xlink:href="{{ ibexa_icon_path('view') }}"></use>
+                    </svg>
+                    <svg class="ibexa-icon ibexa-icon--view-hide">
+                        <use xlink:href="{{ ibexa_icon_path('view-hide') }}"></use>
+                    </svg>
+                    <input
+                        class="ez-checkbox-icon__checkbox"
+                        {{ not location.explicitlyHidden ? 'checked' }}
+                        type="checkbox"
+                        name="updateVisibility"
+                        {% if not can_hide[location.id] %} disabled="disabled"{% endif %}
```

Comment:
similar to https://github.com/ezsystems/ezplatform-admin-ui/pull/1831/files#diff-ecb99eb5e631b447cc39b2a6a4d818d0e13d92ffa6df407bc115767b20392c8bR45, just `{{ not can_hide[location.id] ? 'disabled' }}` ?